src/utilities/color-types.ts

```diff
@@ -36,3 +36,7 @@ export interface HSLAColor extends HSLColor {
   /** Level of transparency */
   alpha: number;
 }
+
+export interface MaybeAlphaHSLAColor extends Omit<HSLAColor, 'alpha'> {
+  alpha?: number;
+}
```

Comment:
This ends up being a better typing for the `hslToString` method, as `alpha` is only required for `hsla` notation.